go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/celestiaorg/celestia-app/v3
 go 1.23.6
 
 require (
-	cosmossdk.io/errors v1.0.1
+	cosmossdk.io/errors v1.0.2
```

Comment:
That [changelog entry](https://github.com/cosmos/cosmos-sdk/blob/errors/v1.0.2/errors/CHANGELOG.md) for v1.0.2 isn't descriptive but I reviewed [the PR](https://github.com/cosmos/cosmos-sdk/pull/23901) and seems safe to bump to.